### ISSUE TITLE
Action space?

### ISSUE THREAD

**Issue by FelixLabelle** [jacobkrantz/VLN-CE#3]
I'm unclear on the action space, specifically the size and significance of the different dimensions of the predicted logits. The size appears to be fixed, but is the meaning absolute (0 is forward, 1 is left, 2 is rights, etc..) or can the meaning change (e.g. for R2R certain agents predict over candidate images).

In the code there are references to the _action_space_ member of _env_, but I'm not too sure how it's initialized. If possible a pointer to the right spot in the code would be greatly appreciated as well!

Edit:

NVM, I took another look at the paper. There are 4 actions:  move forward 0.25m, turn-left or turn-right 15 degrees, stop.
